ResearchKit/Common/ORKTaskViewController.h

```diff
@@ -487,6 +487,13 @@ ORK_CLASS_AVAILABLE
  */
 @property (nonatomic, readonly) UINavigationBar *navigationBar;
 
+/**
+ A Boolean value indicating whether the task can be canceled directly, without prompting the "End Task" action sheet.
+ 
+ The default value of this property is `NO`. To allow the task to be canceled directly, set the value to `YES`.
+ */
+@property (nonatomic, assign) BOOL cancelable;
+
```

Comment:
worth renaming this to "discardable"?

Reply:
I am open to either option, but my concern is whether naming it "discardable" would confuse this with "discarding task results"; wherein this property is for dismissing a `ORKTaskViewController`?

Maybe something like `dismissWithoutPrompt`...?

Reply:
Per your current code path, when the user hits "Cancel" the developer will receive the taskVC: didFinishWithReason: callback with value ORKTaskViewControllerFinishReasonDiscarded, which is the same as "discard results". Which is why I suggested "discardable".

Let me know if that makes sense..

Reply:
Oh good point! That makes perfect sense to me now.